packages/core/src/command/rate-limiting/shared.ts

```diff
@@ -0,0 +1,20 @@
+import { Key } from "aws-sdk/clients/dynamodb";
+import { RateLimitOperation, RateLimit } from "@metriport/shared/src/domain/rate-limiting";
+
+export const secondGranularityIsoDateTime = "YYYY-MM-DDTHH:mm:ss";
+
+type LimitSecondsMap = Map<RateLimit, number>;
+
+export const secondsLookup: LimitSecondsMap = new Map([["operationsPerMinute", 60]]);
+
+type LimitMap = Map<RateLimit, number>;
+
+export const defaultPatientCreateLimits: LimitMap = new Map([["operationsPerMinute", 4]]);
```

Comment:
OOC why did we set 4 as the limit we can create in 60 seconds?

Reply:
I need to a bit of analysis to decide on the exact limit, basically it's a place holder for now and it's what the patient import pace is set at right now.